systems/controllers/iiwa_velocity_controller.cc

```diff
@@ -0,0 +1,78 @@
+#define NUM_JOINTS 7
+#define K_D 25
+#define K_R 3
+
+#include "systems/controllers/iiwa_velocity_controller.h"
+
+namespace dairlib{
+namespace systems{
+
+KukaIiwaVelocityController::KukaIiwaVelocityController()
```

Comment:
Like with the position controller, all of these constants should be input parameters to the constructor.

Also as with the position controller, I think the paired input/output ports for position/velocity should be combined, unless there's a good reason not to. This will simplify the code noticeably.